sympy/polys/multivariate_resultants.py

```diff
@@ -110,44 +114,50 @@ def get_dixon_polynomial(self):
             rows.append([f.subs(substitution) for f in self.polynomials])
 
         A = Matrix(rows)
-        product_of_differences = Mul(*[a - b for a, b in zip(self.variables,
-                                                         self.dummy_variables)])
+
+        terms = zip(self.variables, self.dummy_variables)
+        product_of_differences = Mul(*[a - b for a, b in terms])
         dixon_polynomial = (A.det() / product_of_differences).factor()
 
         return poly_from_expr(dixon_polynomial, self.dummy_variables)[0]
 
     def get_upper_degree(self):
-        list_of_products = [self.variables[i] ** ((i + 1) * self.max_degrees[i] - 1)
-                            for i in range(self.n)]
+        list_of_products = [self.variables[i] ** ((i + 1) * \
+                            self.max_degrees[i] - 1) for i in range(self.n)]
```

Comment:
There is no need for backslash continuation as everything is surrounded by brackets.

PEP 8:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.